ksql-engine/src/main/java/io/confluent/ksql/function/udf/math/Sign.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf.math;
+
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
+import io.confluent.ksql.util.KsqlConstants;
+
+@SuppressWarnings("WeakerAccess") // Invoked via reflection
+@UdfDescription(
+    name = "sign",
+    author = KsqlConstants.CONFLUENT_AUTHOR,
+    description = "The sign of a value."
+)
+public class Sign {
+  @Udf(description = "Returns the sign of an INT value.")
+  public Double sign(
```

Comment:
seems very strange to return Doubles from these when the return value can,  by definition, only be integral ? this kind of counter-intuitive behavior will trip up users